internal/controller/group_controller.go

```diff
@@ -65,12 +65,22 @@ func (r *GroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		"request": req.NamespacedName.String(),
 	})
 
+	var isError = false
```

Comment:
instead of setting `isError` to `false` here, can we mark this as false at the beginning for the for loop else it is going to remain `true` for all the backends in the case first backend reconciliation fails.